tools/json2json.cpp

```diff
@@ -37,7 +37,7 @@ int main(int argc, char *argv[]) {
   if(!result.count("file")) {
     std::cerr << "No filename specified." << std::endl;
     std::cerr << options.help() << std::endl;
-    exit(1);
```

Comment:
Regarding the syntax "exit(1)", please see https://stackoverflow.com/questions/13667364/exit-failure-vs-exit1

Reply:
Thanks, I'm gonna replace it in all the code